shared/vue-ui/src/molecules/AbcdeScore/AbcdeScore.vue

```diff
@@ -1,19 +1,31 @@
 <script setup>
-defineProps({
+import { computed } from 'vue';
+
+const props = defineProps({
   value: {
     type: String,
-    required: true,
+    default: undefined,
     validator(value) {
-      return ['A', 'B', 'C', 'D', 'E'].includes(value)
+      return value === undefined || ['A', 'B', 'C', 'D', 'E'].includes(value)
```

Comment:
nitpick: even if it's not a string, it would probably be cleaner to put the accepted undefined value in the array of accepted values

`['A', 'B', 'C', 'D', 'E', undefined].includes(value)`